server/resource/v1/kv_resource.go

```diff
@@ -245,16 +244,13 @@ func (r *KVResource) URLPatterns() []restful.Route {
 			Produces: []string{goRestful.MIME_JSON},
 		}, {
 			Method:           http.MethodDelete,
-			Path:             "/v1/kv/{ids}",
+			Path:             "/v1/kv/",
```

Comment:
Path:             "/v1/kv/", 

since param is path parameters, I guess you shoult write this Path right